backend/tests/dining/test_views.py

```diff
@@ -45,16 +93,13 @@ def test_get(self):
         response = self.client.get(reverse("venues"))
         for entry in response.json():
             self.assertIn("name", entry)
-            self.assertIn("address", entry)
             self.assertIn("days", entry)
             self.assertIn("image", entry)
             self.assertIn("id", entry)
             for day in entry["days"]:
                 self.assertIn("date", day)
-                self.assertIn("status", day)
```

Comment:
Hm, can you double check with iOS and Android (in the f22 channel) to make sure they don't use this `status` variable? Preferably this PR will be backwards compatible. I think this `status` just shows "open/closed" but that can easily be derived by them by just checking the start and end times. (same with `address`)

Reply:
ok. confirmed and we should be good.